### ISSUE TITLE
syslog `octet_count` not compliant with RFC

### ISSUE THREAD

**Issue by marojor** [fluent/fluentd#2982]
**Describe the bug**

Looking at the doc for syslog, `frame_type` [section](https://docs.fluentd.org/input/syslog#frame_type):

```44 <6>Sep 10 00:00:00 localhost logger: hello!```

The [RFC-6587](https://tools.ietf.org/html/rfc6587#section-3.4.1) defines the syslog frame as:

```SYSLOG-FRAME = MSG-LEN SP SYSLOG-MSG```

But in the example the length of the `SYSLOG-MSG` is 43:

```
>>> len("""<6>Sep 10 00:00:00 localhost logger: hello!""")
43
```

It looks like the space is being counted even when the RFC explicity says it should not:

```
MSG-LEN is the octet count of the SYSLOG-MSG in the SYSLOG-FRAME.
```

The implementation works consistently with the documentation.

**Expected behavior**

For the given example, the MSG-LEN should be 43 (space should be ignored).

**Comment by ganmacs** [comment]
First, please follow the template.
also, fluentd does just parse the input syslog. so it never creates `44 <6>Sep 10 00:00:00 localhost logger: hello!`. plus, you don't provide the reproduction step. so we can't handle it.

**Comment by repeatedly** [comment]
Agreed. I tested `43 <6>Sep 10 00:00:00 localhost logger: hello!43 <6>Sep 10 00:00:00 localhost logger: hello!` with in_syslog and it worked.
Could you show example log to reproduce the bug?
or documentation mistake?

**Comment by marojor** [comment]
@repeatedly 

I found the bug using 1.9.3 (plugin was _eating_ the last byte), but the issue seems to be solved in the code with 1.10.3.

I've opened https://github.com/fluent/fluentd-docs-gitbook/pull/174 to fix the documentation

@ganmacs Thanks for the feedback, I'll follow the template next time!
